src/chapters/jdbc-auth.xml

```diff
@@ -299,6 +343,61 @@ Type "help" for help.
+<prompt>2></prompt> <userinput>GO</userinput>
+<prompt>1></prompt> <userinput>ALTER ROLE db_datawriter ADD MEMBER <replaceable>guacamole_user</replaceable>;</userinput>
+<prompt>2></prompt> <userinput>ALTER ROLE db_datareader ADD MEMBER <replaceable>guacamole_user</replaceable>;</userinput>
+<prompt>3></prompt> <userinput>GO</userinput></screen>
+            </informalexample>
+            <para>Once the database and user account is created, and the user associated with the database, you can use
+                the supplied scripts to load the schema into the database.  These scripts are included in the
+                <filename>sqlserver/schema/</filename> directory of the archive you downloaded from the Guacamole
+                web site.</para>
+            <informalexample>
+                <screen><prompt>$</prompt> <userinput>/opt/mssql-tools/bin/sqlcmd -S localhost -U <replaceable>guacamole_user</replaceable> -d <replaceable>guacamole_db</replaceable> -i schema/001-create-schema.sql</userinput>
+<prompt>Password:</prompt> <userinput><replaceable>some_password</replaceable></userinput>
+<computeroutput>Rule bound to data type.
+The new rule has been bound to column(s) of the specified user data type.
+Rule bound to data type.
+The new rule has been bound to column(s) of the specified user data type.</computeroutput>
+<prompt>$</prompt> <userinput>/opt/mssql-tools/bin/sqlcmd -S localhost -U <replaceable>guacamole_user</replaceable> -d <replaceable>guacamole_db</replaceable> -i schema/002-create-admin-user.sql</userinput>
+<prompt>Password:</prompt> <userinput><replaceable>some_password</replaceable></userinput>
+<computeroutput>
+(1 rows affected)
+
+(3 rows affected)
+                        
+(5 rows affected)</computeroutput></screen>
+            </informalexample>
+            <para>If the operation is successful, the tables and permissions have been created successfully, and you
+                can now use the database with the Guacamole client web application.</para>
+            <important xml:id="jdbc-auth-sqlserver-upgrade">
+                <para>If you are upgrading from an older version of Guacamole and were already using
+                    SQL Server, you may need to run one or more database schema upgrade scripts
+                    located within the <filename>schema/upgrade/</filename> directory. Each of these
+                    scripts is named
+                        <filename>upgrade-pre-<replaceable>VERSION</replaceable>.sql</filename>
+                    where <replaceable>VERSION</replaceable> is the version of Guacamole where those
+                    changes were introduced. They need to be run when you are upgrading from a
+                    version of Guacamole older than <replaceable>VERSION</replaceable>.</para>
+                <para>SQL Server support was added in version <replaceable>VERSION</replaceable>, so, at the time of this
```

Comment:
The wording here is odd, referring to the specific version where SQL Server support was added with the placeholder that is meant to refer to whichever version is current:

![added-in-version-version](https://user-images.githubusercontent.com/4632905/34449767-fed1ad64-ecb0-11e7-8159-3e0e2494a579.png)

If the `<important>` section is just removed overall, however, that that will address this, too.

Reply:
Removed.